packages/sds/src/message_channel/repair/buffers.ts

```diff
@@ -0,0 +1,280 @@
+import { Logger } from "@waku/utils";
+
+import type { HistoryEntry } from "../message.js";
+
+const log = new Logger("sds:repair:buffers");
+
+/**
+ * Entry in the outgoing repair buffer with request timing
+ */
+interface OutgoingBufferEntry {
+  entry: HistoryEntry;
+  tReq: number; // Timestamp when this repair request should be sent
+  requested: boolean; // Whether this repair has been requested already
```

Comment:
```suggestion
  requested: boolean; // Whether this repair has already been requested by the local node
```

right? I think better be explicit.